package.json

```diff
@@ -12,20 +12,16 @@
     "react-dom": "^16.6.0"
   },
   "devDependencies": {
-    "babel-core": "^6.26.0",
+    "@babel/core": "^7.2.2",
+    "@babel/preset-env": "^7.2.0",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "9.0.0",
     "babel-jest": "^23.4.0",
-    "babel-plugin-external-helpers": "^6.22.0",
-    "babel-plugin-transform-flow-strip-types": "^6.22.0",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-1": "^6.24.1",
-    "benchmark": "^2.1.4",
     "codecov": "^3.0.2",
     "concurrently": "^3.5.1",
     "danger": "6.1.12",
-    "enzyme": "^3.5.0",
```

Comment:
@chrislloyd seems like this is breaking the build - any reason for removing `enzyme`?